src/shogun/mathematics/RandomNamespace.h

```diff
@@ -22,9 +26,8 @@ namespace shogun
 		 *
 		 * @param container the container holding the elements
 		 */
-		template <
-		    template <typename> class Container, typename T, typename PRNG>
-		static inline void shuffle(Container<T>& container, PRNG& prng)
+		template <typename Container, typename PRNG>
+		static inline void shuffle(Container& container, PRNG& prng)
```

Comment:
do we really need this? i mean `std::shuffle` is basically doing this for you... or you just wanna have a syntactic sugar?

Reply:
In the case this grows and checks for gpu backend. It's not really useful otherwise